src/pathfora.js

```diff
@@ -2824,10 +2824,32 @@
      * @throws {Error} error
      * @param {string} accountId  Lytics account ID
      */
-    recommendContent: function (accountId, params, callback) {
+    recommendContent: function (accountId, params, id, callback) {
       // Recommendation API:
       // https://www.getlytics.com/developers/rest-api#content-recommendation
 
+      // if we have the recommendation response cached in session storage
+      // use that instead of making a new API request
+      var storedRec = sessionStorage.getItem('PathforaRecommend_' + id);
```

Comment:
we should probably store the recommend prefix as a constant and reference that constant throughout the library.